### DISCUSSION TITLE
Two suggestions: detailed demo document and add I / O setting button to send commands to MCU

### DISCUSSION THREAD

**Discussion by zhigangbox** [Serial-Studio/Serial-Studio#54] [Ideas]
First of all, thank the author for providing such a good open source software.

But I have two suggestions:

1. Make a detailed demo document description: step by step, first let the novice run the demo and see the corresponding relationship between the serial data and visualization; Then users can learn how to modify the visual display variables, and then adjust the sending content, which can be used in practice.

I'm not sure about the correspondence and revision. I think other newcomers are like me.

2 add I / 0 command button: at present, the software only displays from MCU to serial port in one direction, but in fact, users have a strong demand from visual I / O component to MCU to send various commands to MCU. It has a simple and efficient analytical method. That would be great.

**Comment by alex-spataru** [comment]
Hi! Both requests are available as of the latest version, please take a look at the new [wiki](https://github.com/Serial-Studio/Serial-Studio/wiki) and the new [Actions](https://github.com/Serial-Studio/Serial-Studio/wiki/Project-Editor#action-view-when-an-action-is-selected) feature.
